static/locales/en-US.yaml

```diff
@@ -920,6 +922,8 @@ Starting download: 'Starting download of "{videoTitle}"'
 Downloading failed: 'There was an issue downloading "{videoTitle}"'
 Screenshot Success: Saved screenshot as "{filePath}"
 Screenshot Error: Screenshot failed. {error}
+Channel Hidden: Channel {channel} added to filter
```

Comment:
I think this messaging can be better now that #4202 is here

I think `{channel} added to Channel filter` is more explicit